django_redis/client/sharded.py

```diff
@@ -238,16 +238,22 @@ def keys(self, search, version=None):
         decoded_keys = (smart_text(k) for k in keys)
         return [self.reverse_key(k) for k in decoded_keys]
 
-    def delete_pattern(self, pattern, version=None, client=None):
+    def delete_pattern(self, pattern, version=None, client=None, count=None):
```

Comment:
Agreed. This is a test i mentioned in the standup. Since fakeredis is already in place the test is going to use it.